models.py

```diff
@@ -386,6 +388,7 @@ def __repr__(self):
     def from_payload(cls, payload: dict, dataset_id: str) -> "Resource":
         data = payload.copy()
         data["resource_id"] = data.pop("id")
+        data["available"] = bool((data.get("extras") or {}).get("check:available"))
```

Comment:
why do we need this here if it's defined in computer?

Reply:
Because I forgot to remove it from computed...